data/net/minecraft/world/level/block/VineBlock.mapping

```diff
@@ -30,6 +30,10 @@ CLASS net/minecraft/world/level/block/VineBlock
 		ARG 4 context
 	METHOD getStateForPlacement (Lnet/minecraft/world/item/context/BlockPlaceContext;)Lnet/minecraft/world/level/block/state/BlockState;
 		ARG 1 context
+	METHOD getUpdatedState (Lnet/minecraft/world/level/block/state/BlockState;Lnet/minecraft/world/level/BlockGetter;Lnet/minecraft/core/BlockPos;)Lnet/minecraft/world/level/block/state/BlockState;
+		ARG 1 state
+		ARG 2 blockGetter
```

Comment:
```suggestion
		ARG 2 level
```

Reply:
I kind of wondered about this. Will change in next commit.